docs/data-engineering-workshop.md

```diff
@@ -0,0 +1,21 @@
+This workshop is designed to show data engineers and system administrators
+how to architect a data infrastructure service using the Open Data Hub. Our
+goal is for engineers who are interested in developing platforms for the ML
+and Analytics needs to walk away having learned data wrangling in a Hybrid
+Cloud environment with Spark and JupyterHub.
+
+Unlike some other workshops, the entirety of the activity is conducted within
+JupyterHub.
+
```

Comment:
I might add a bunch of links to this section (open data hub, jupyter, jupyterhub, spark) in case the user got here through discovery on demo.os.c vs getting linked to here